runtime/js-compute-runtime/builtins/shared/console.cpp

```diff
@@ -495,12 +491,11 @@ static bool count(JSContext *cx, unsigned argc, JS::Value *vp) {
   std::string label = "";
   if (args.hasDefined(0)) {
     auto label_val = args.get(0);
-    size_t length;
-    auto label_string = encode(cx, label_val, &length);
+    auto label_string = fastly::core::encode(cx, label_val);
     if (!label_string) {
       return false;
     }
-    label = label_string.get();
+    label = std::string{label_string.begin(), label_string.len};
```

Comment:
Would this be a copy?

Reply:
Yes, every place we use a `std::string` it'll be a copy. I think we can refactor this to avoid the copies, but that seemed like a good change for a separate PR.

Reply:
Also this isn't introducing a new copy, just being explicit about the construction of the intermediate string -- the assignment of a char pointer to a string will copy the contents of the char pointer into the string destination.